npdarrington/fitlit
# Nd refactor

What’s this PR do?  
- This PR is a refactor of all classes to make methods that are DRY and SRP.
- Also refactored inconsistent styling to match eslint rules.
- Added some helper methods in the Sleep class to help with return all users over 3 sleep quality average.
- Deleted some redundant code and created new methods to get all sleep data and display the data based on parameter.
 
Where should the reviewer start? 
- Git pull these updates.
 
How should this be manually tested?  
- npm test to make sure all tests are still passing
- Open index.html in the browser.
 
Any background context you want to provide?  
- Some method calls in script.js were not changed in accordance with the new sleep methods due to Taylor working on getting the Activity DOM displaying. These changes will be implemented in the next branch after merging to avoid conflicts.
 
What are the relevant tickets?  
#55 
 
Screenshots (if appropriate)  
- N/A 

Questions: 
- N/A
